src/Approximations/overapproximate.jl

```diff
@@ -1736,3 +1736,50 @@ function overapproximate(CHA::ConvexHullArray{N, <:AbstractZonotope{N}}, ::Type{
         return Zaux
     end
 end
+
+"""
+    overapproximate(Z::Zonotope, ::Type{<:HParallelotope}, indices=1:dim(Z))
+
+Overapproximation of a zonotope with a parallelotopic set in constraint
+representation.
+
+### Input
+
+- `Z` -- zonotope
+- `HParallelotope` -- type for dispatch
+- `indices` -- (optional; default: `1:dim(Z)`) generator indices selected when
+                constructing the parallelotope
+
+### Output
+
+An overapproximation of the given zonotope using a parallelotope.
+
+### Algorithm
+
+The algorithm is based on Proposition 8 discussed in Section 5 of [1].
+
+[1] Althoff, M., Stursberg, O., & Buss, M. (2010). *Computing reachable sets of
+hybrid systems using a combination of zonotopes and polytopes*. Nonlinear
+analysis: hybrid systems, 4(2), 233-249.
+"""
+function overapproximate(Z::Zonotope, ::Type{<:HParallelotope}, indices=1:dim(Z))
+    Zred = _overapproximate_hparallelotope(Z, HParallelotope, indices)
+    return convert(HParallelotope, Zred)
+end
+
+function _overapproximate_hparallelotope(Z::Zonotope, ::Type{<:HParallelotope}, indices=1:dim(Z))
```

Comment:
What is the purpose of the second argument, given the function name?

Reply:
if you want to propose another name that's fine! also, i don't mind to remove the second argument as its implicitly used.

Reply:
I would remove it.

Reply:
done!